sqlalchemy_redshift/dialect.py

```
@@ -1227,20 +1412,35 @@ def create_connect_args(self, *args, **kwargs):
+    def dbapi(cls):
+        # Backwards compatibility with SQLAlchemy < 2.0
+        return cls.import_dbapi()
+
 
 class RedshiftDialect_psycopg2(
     Psycopg2RedshiftDialectMixin, PGDialect_psycopg2
 ):
     supports_statement_cache = False
 
+    @classmethod
+    def import_dbapi(cls):
```

Ruff (0.14.0)

Missing return type annotation for classmethod `import_dbapi`

(ANN206)